### PR TITLE
Upgrade to webpack 2

### PR DESCRIPTION
**Pull request by GeKorm**
Addresses #157

Only a couple changes to webpack.config.js were required.

5bd5ec3093a8908ac3adc8224d92846836bb30a1 Isn't necessary, just showing that updated dependencies work, especially babel-loader 7.0.0

### CODE REVIEW

**Comment by pburtchaell** [comment]
Thanks @GeKorm! It looks like CI is failing. [See the logs](https://travis-ci.org/pburtchaell/redux-promise-middleware/builds/241979855?utm_source=github_status).

```
$ yarn
yarn install v0.24.6
[1/4] Resolving packages...
[2/4] Fetching packages...
error webpack@2.6.1: The engine "node" is incompatible with this module. Expected version ">=4.3.0 <5.0.0 || >=5.10".
error Found incompatible module
```

Based off the error, I we'll need to upgrade the [Node version](https://github.com/pburtchaell/redux-promise-middleware/blob/master/.travis.yml#L3).

**Comment by GeKorm** [comment]
@pburtchaell Thanks! I hope this will do.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - "4.1"
```
Comment (pburtchaell) [inline .travis.yml]: Can you actually just update to `"6"`?

**Comment by pburtchaell** [comment]
Great! I need to resolve #159 before this is merged, but I will get to it as soon as possible. 😄

**Pull request merged**